### ISSUE TITLE
Strange behaviour on send

### ISSUE THREAD

**Issue by zathras-crypto** [zathras-crypto/omnicore#71]
Noticed during testing this morning - transaction sat in unconfirmed state for a few minutes before transitioning to "Offline".  No conflicts that I can see and client had 8 connections at time of send.

![omni1](https://cloud.githubusercontent.com/assets/5633351/7716709/31d47b92-fed9-11e4-857b-3c263d290dc5.png)

**Comment by dexX7** [comment]
@zathras-crypto: this is very likely the bug where unconfirmed transactions are added out of order back to the memory pool after a restart.

The fix was merged upstream, so we may adopt it: https://github.com/bitcoin/bitcoin/pull/5511

**Comment by zathras-crypto** [comment]
There was no restart though?  The client was running the whole time when it transitioned from "Unconfirmed" to "Offline".

EDIT: I think - perhaps I'm getting myself confused now :( 

Running a secondary test now

EDIT: secondary test was fine - I'm almost certain I didn't close the client though because the trade details for that transaction were still populated in the exchange page...  For now let's put on hold unless I can reproduce.

**Comment by dexX7** [comment]
Ah sorry, I just noticed it's "offline", and not "conflicting".

If I recall correctly, then this can be reproduced by flooding the client with many, many transactions. I'm not 100 % sure anymore how, but if I'm not mistaken, then it's a Bitcoin Core issue, and not caused by our changes.

Edit: hmm.. even after running the tests a few times, everything is fine.

**Comment by zathras-crypto** [comment]
Yeah it was really strange behaviour - the transaction sat unconfirmed for a few minutes, then went to "offline" until it got confirmed, and then got parsed and processed as normal.

I've seen loads of conflicts before across several of my testing wallets but this is almost universally down to closing and reopening the client which sometimes allows the client to re-use the same inputs (likely due to the out of order mempool issue you noted) - I'm near certain this is a different issue and it's the first time I've seen such behaviour but I haven't been able to reproduce it again.
